tests/datasource_userlist_test.php

```diff
@@ -117,7 +122,7 @@ public function test_config_delete() {
         global $DB;
 
         $manager = new manage_content_types();
-        $cmstype = $manager->create((object) ['name' => 'Name', 'idnumber' => 'test-name']);
+        $cmstype = $manager->create((object) ['name' => 'Name', 'idnumber' => 'test-name', 'datasources' => 'list']);
```

Comment:
To keep it consistent with the class and outward facing name, shouldn't this be `userlist` instead of `list`?

Reply:
So we're just going to leave this for now?